### PR TITLE
[core] Connection between ui and python

### PR DESCRIPTION
**Pull request by KlicOgogo**
- core/ and data/ for .py and .csv respectively;
- get_ratings dumps to rating.json;
- get_ratings dumps ratings history to ratings_history.json;
- data with telegram usernames;
- refactoring;

### CODE REVIEW

File: core/logger.py [hunk 1]
```diff
@@ -55,8 +58,8 @@ def score_is_correct(score):
 
 
 def main(_):
-    games_scv_filename = 'games.csv'
-    games = pd.read_csv(games_scv_filename)
+    games_scv_filepath = os.path.join(repo_root_dir, 'data/games.csv')
```
Comment (Drapegnik) [inline core/logger.py]: maybe `games_csv_filepath`?

File: core/get_ratings.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+def update_ratings_with_game(ratings, game):
+    winner = game.Winner
+    looser = game.Looser
+
+    sets = list(map(int, game.Score.split(':')))
+
+    winner_sets = np.max(sets)
+    looser_sets = np.min(sets)
+
+    ball_coeffs = {'blue': 0.5, 'red': 0.7, 'yellow': 1.0}
```
Comment (Drapegnik) [inline core/get_ratings.py]: maybe extract to constant `BALL_COEFFS` and move to `core/constants.py`?

File: core/get_ratings.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    return new_ratings
+
+
+def count_ratings(date):
+    game_data_filepath = os.path.join(repo_root_dir, 'data/games.csv')
+    games = pd.read_csv(game_data_filepath)
+    sorted_games = games.sort_values(['Date', 'Game of the day'])
+    proper_games = sorted_games[sorted_games['Date'] <= date]
+    all_players = list(set(proper_games['Winner'].tolist() + proper_games['Looser'].tolist()))
+    data = {'Rating': [1400.00 for i in range(len(all_players))]}
```
Comment (Drapegnik) [inline core/get_ratings.py]: extract `1400.00` to `DEFAULT_RATING`

File: core/get_ratings.py [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+    json_filename = os.path.join(repo_root_dir, 'leaderboard-ui/src/rating.json')
+    with open(json_filename, 'w') as json_file:
+        json.dump(json_list, json_file, indent=2)
+        json_file.write('\n')
+
+
+def save_ratings_history_to_json(ratings_log):
+    json_list = []
+    for ratings in ratings_log:
+        ratings_dict = {name: int(np.round(ratings[1].loc[name]['Rating'])) for name in ratings[1].index}
```
Comment (Drapegnik) [inline core/get_ratings.py]: `int(np.round(ratings[1].loc[name]['Rating'])` - scary

could you use unzip in loop? like:

```py
for (date, data) in ratings_log:
```

File: core/get_ratings.py [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+        json_file.write('\n')
+
+
+def main(_):
+    if FLAGS.day is None:
+        cur_date = datetime.now()
+        cur_year = str(cur_date.year)
+        cur_month = '0' * (cur_date.month < 10) + str(cur_date.month)
+        cur_day = '0' * (cur_date.day < 10) + str(cur_date.day)
+        cur_day_of_year = '{}-{}-{}'.format(cur_year, cur_month, cur_day)
```
Comment (Drapegnik) [inline core/get_ratings.py:116]:
```suggestion
        cur_day_of_year = f'{cur_year}-{cur_month}-{cur_day}'
```

**Review by Drapegnik — CHANGES_REQUESTED** [review]

File: core/get_ratings.py (hunk 4 above)
Comment (KlicOgogo) [reply core/get_ratings.py]: `int(np.round())` is the best way to do that. .___.

File: core/get_ratings.py (hunk 5 above)
Comment (KlicOgogo) [reply core/get_ratings.py:116]: Too new feature

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**